Backend/core/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "chase-car-dashboard-backend-core"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
```

Comment:
This one should have an author and a description. In the description, explain why this is broken into a separate pyproject.toml from the rest of the backend